doc/classes/Physics2DServer.xml

```diff
@@ -654,7 +654,19 @@
 			<return type="void" />
 			<argument index="0" name="rid" type="RID" />
 			<description>
-				Destroys any of the objects created by Physics2DServer. If the [RID] passed is not one of the objects that can be created by Physics2DServer, an error will be sent to the console.
+				Destroys an object created by the Physics2DServer. If the [RID] passed is not one created by the Physics2DServer, an error will be sent to the console.
+				Note: After freeing the object, the RID now has a reference to invalid memory. It is not safe to free an invalid RID. Make sure to reset it before using it again by assigning it to `RID()`.
+				[code]
```

Comment:
These should be `[codeblock]`, not `[code]`. `[code]` is for inline code `like this`.

BTW to reduce redundancy, maybe you could add the detailed explanations and example in VisualServer only, and then add a reference to it in the physics and navigation servers? Like:
```
[b]Note:[/b] See [method VisualServer.free_rid] for details on how to handle RIDs for freed objects.
```

Reply:
done. also for navserver docs.